libpod/runtime_volume_linux.go

```diff
@@ -67,9 +67,13 @@ func (r *Runtime) newVolume(ctx context.Context, options ...VolumeCreateOption)
 	if volume.config.Driver == define.VolumeDriverLocal {
 		logrus.Debugf("Validating options for local driver")
 		// Validate options
-		for key := range volume.config.Options {
+		for key, val := range volume.config.Options {
 			switch key {
-			case "device", "o", "type", "UID", "GID", "SIZE", "INODES":
+			case "device":
+				if _, err := os.Stat(val); err != nil {
```

Comment:
I think this might be undesirable - e.g. for `tmpfs` device can be set to anything, so long as the destination path is valid